web/layouts/404.html

```diff
@@ -4,10 +4,17 @@ <h2>Page not found!</h2>
 
 <p>We are sorry, but this page does not exist.
 <script>
-    if (document.location.toString().includes("/master/")) {
-        document.write("This page has been <a href=\"")
-        document.write(document.location.toString().replace("/master/", "/nightly/"))
-        document.write("\">moved</a>.")
+    var loc = document.location.toString();
+    if (loc.includes("/master/") || loc.includes("/index.html")) {
+        newLocation = document.location.toString()
+                .replace("/master/", "/nightly/")
+                .replace("/index.html", "/index-el.html");
+        document.write("This page has been <a href=\"");
+        document.write(newLocation);
+        document.write("\">moved</a>. Redirecting in 3 seconds.");
```

Comment:
you can do string interpolation like: 
```js
`This page has been moved to ${newlocation}. Redirecting in 3 seconds`
```

Reply:
Thanks, these are all valid points but what I have is kind-of-a tested and since everyone is okay I am gonna merge this to shorten the time we don't have this in our 404 page. I learned a lot, cheers! :-)